cfy_manager/components/restservice/restservice.py

```diff
@@ -113,7 +114,7 @@ def _configure_dbus():
             ln(source=dbus_bindings, target=dbus_venv_path, params='-sf')
         if not islink(join(REST_VENV, site_packages)):
             ln(source=dbus_glib_bindings, target=join(
-                    REST_VENV, site_packages), params='-sf')
+               REST_VENV, site_packages), params='-sf')
```

Comment:
my pylint is a bit more strict than the one we use in circle, lemme be a bit ocd here :P